src/models/room.ts

```diff
@@ -55,8 +55,8 @@ import { TypedEventEmitter } from "./typed-event-emitter";
 // room versions which are considered okay for people to run without being asked
 // to upgrade (ie: "stable"). Eventually, we should remove these when all homeservers
 // return an m.room_versions capability.
-const KNOWN_SAFE_ROOM_VERSION = '6';
-const SAFE_ROOM_VERSIONS = ['1', '2', '3', '4', '5', '6'];
+const KNOWN_SAFE_ROOM_VERSION = '9';
```

Comment:
> In practice, KNOWN_SAFE_ROOM_VERSION should be the same as the common default room version

Hmm @turt2live you sure about this given the spec is still saying default to `6`

Reply:
Just missing spec at the moment: https://github.com/matrix-org/matrix-spec-proposals/pull/3739